src/plugins/data/common/search/aggs/buckets/date_range_fn.ts

```diff
@@ -91,7 +91,7 @@ export const aggDateRange = (): FunctionDefinition => ({
     json: {
       types: ['string'],
       help: i18n.translate('data.search.aggs.buckets.dateRange.json.help', {
-        defaultMessage: 'Advanced json to include when the agg is sent to Elasticsearch',
```

Comment:
How did we miss this in past audits? Do we need to re-open an issue to search for strings?

Reply:
I dont think we do. I just noticed them when outputting strings and they were the only bad ones.